Sources/MongoSwift/MongoClient.swift

```diff
@@ -352,14 +353,17 @@ public class MongoClient {
      *    - `LogicError` if the provided session is inactive.
      *    - `LogicError` if this client has already been closed.
      *    - `EncodingError` if an error is encountered while encoding the options to BSON.
+     *    - `CommandError` when options.authorizedDatabases is false:
```

Comment:
sorry to be pedantic...

1. can we phrase this more similarly to the previous bullet points? by that I mean the "`CommandError` if ...".  format. so something like "if options.authorizedDatabases is false and the user does not have listDatabases permissions."

2. listDatabase -> listDatabases